src/sentry/static/sentry/app/components/stream/groupRowActions.tsx

```diff
@@ -0,0 +1,145 @@
+        data,
+        query,
+        environment: selection.environments,
+        ...projectConstraints,
+        ...selection.datetime,
+      },
+      {
+        complete: () => {
+          clearIndicators();
+        },
+      }
+    );
+  };
+
+  render() {
+    const {orgId, group} = this.props;
+
+    return (
+      <Wrapper>
+        <HoverActionsWrapper>
+          <ActionWrapper>
+            <Tooltip title={t('Move to Backlog')}>
+              <ActionLink
+                className="btn btn-default btn-sm"
+                onAction={() => this.handleUpdate({inbox: false})}
+                shouldConfirm={false}
+                title={t('Move to backlog')}
+              >
+                <IconIssues size="xs" color="gray300" />
+              </ActionLink>
+            </Tooltip>
+          </ActionWrapper>
+
+          <ActionWrapper>
+            <Tooltip title={t('Ignore')}>
+              <IgnoreActions
+                onUpdate={this.handleUpdate}
+                shouldConfirm={false}
+                inboxRowAction
+              />
+            </Tooltip>
+          </ActionWrapper>
+
+          <ActionWrapper>
+            <Tooltip title={t('Resolve')}>
+              <Projects orgId={orgId} slugs={[group.project.slug]}>
+                {({projects, initiallyLoaded, fetchError}) => {
+                  const project = projects[0];
+                  return (
+                    <ResolveActions
+                      hasRelease={
+                        project.hasOwnProperty('features')
+                          ? (project as Project).features.includes('releases')
+                          : false
+                      }
+                      latestRelease={
+                        project.hasOwnProperty('latestRelease')
+                          ? ((project as Project).latestRelease as Release)
+                          : undefined
+                      }
+                      orgId={orgId}
+                      projectId={group.project.id}
+                      onUpdate={this.handleUpdate}
+                      shouldConfirm={false}
+                      inboxRowAction
+                      disabled={!!fetchError}
+                      disableDropdown={!initiallyLoaded || !!fetchError}
+                      projectFetchError={!!fetchError}
+                    />
+                  );
+                }}
+              </Projects>
+            </Tooltip>
+          </ActionWrapper>
+        </HoverActionsWrapper>
+      </Wrapper>
+    );
+  }
+}
+
+const Wrapper = styled('div')`
+  position: absolute;
+  background: transparent;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+`;
+
+const HoverActionsWrapper = styled('div')`
+  width: 200px;
+  height: 100%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-end;
+`;
+
+const ActionWrapper = styled('div')`
```

Comment:
imo, you don't really need `Wrapper` here

Reply:
I need it for the margin right

Reply:
Actually I can use `space-between` instead